### PR TITLE
add graphvis DOT files to dump mir directory

### PR DESCRIPTION
**Pull request by mikhail-m1**
r? @nikomatsakis

### CODE REVIEW

File: src/librustc/session/config.rs [hunk 1]
```diff
@@ -1057,6 +1057,8 @@ options! {DebuggingOptions, DebuggingSetter, basic_debugging_options,
           "dump MIR state at various points in translation"),
     dump_mir_dir: Option<String> = (None, parse_opt_string, [UNTRACKED],
           "the directory the MIR is dumped into"),
+    dump_mir_graphviz: bool = (false, parse_bool, [UNTRACKED],
+          "additionally to MIR create graphviz DOT files"),
```
Comment (nikomatsakis) [inline src/librustc/session/config.rs]: Maybe: when dumping MIR, also create graphviz-style `.dot` files

**Review by nikomatsakis — CHANGES_REQUESTED** [review]

**Comment by nikomatsakis** [comment]
Looks good! Left a small comment.

**Comment by kennytm** [comment]
@nikomatsakis The current wording is 

> additionally to `.mir` files, create graphviz `.dot` files

is it good enough for an r=?

**Review by nikomatsakis — APPROVED** [review]

**Comment by kennytm** [comment]
@bors retry 

* Travis incident causing Linux job not starting. https://www.traviscistatus.com/incidents/4vdl52d28hz3

**Pull request merged**